README.md

```diff
@@ -48,7 +50,7 @@ The following example takes you through a scenario where:
 
 1. Compile and build a release node
 ```bash
-cargo build --release
+cargo +nightly-2020-10-05 build --release
```

Comment:
Consider using a `rust-toolchain` file.

docs: https://rust-lang.github.io/rustup/overrides.html?highlight=rust-toolchain#the-toolchain-file
example: https://github.com/PureStake/moonbeam/blob/master/rust-toolchain